include/abb_librws/rws_client.h

```diff
@@ -574,6 +574,16 @@ class RWSClient : public POCOClient
    * \return RWSResult containing the result.
    */
   RWSResult endSubscription();
+
+  /**
+   * \brief Close the active subscription connection.
+   *
+   * This will cause waitForSubscriptionEvent() to return or throw.
+   * It does not delete the subscription from the controller.
+   *
+   * This function blocks until an active waitForSubscriptionEvent() has finished.
+   */
+  void closeSubscription();
```

Comment:
```suggestion
  void forceCloseSubscription();
```
Slightly more descriptive name.